pkg/apis/autoscaling/v1alpha1/pa_validation_test.go

```diff
@@ -124,19 +124,19 @@ func TestPodAutoscalerSpecValidation(t *testing.T) {
 			ProtocolType: net.ProtocolHTTP1,
 		},
 		want: apis.ErrOutOfBoundsValue(-1, 0,
-			v1beta1.RevisionContainerConcurrencyMax, "containerConcurrency"),
```

Comment:
@mattmoor Since we are getting rid of type `RevisionContainerConcurrencyType` and the validation on the type, we no longer get error specifying the field `containerConcurrency`. Is that okay?